apps/web/composables/useBlockManager/useBlockManager.ts

```diff
@@ -0,0 +1,75 @@
+import { ref, computed, onMounted } from 'vue';
+import { useHomepage } from '~/composables/useHomepage/useHomepage';
+import { Block } from '~/composables/useHomepage/types';
```

Comment:
Nuxt's auto-import should take care of all imports from composables and vue. We shouldn't have to declare them explicitly (same for `index.vue`).